hack/verify-golint.sh

```diff
@@ -63,13 +67,13 @@ for p in "${all_packages[@]}"; do
   # completely.
   # Ref: https://github.com/kubernetes/kubernetes/pull/67675
   # Ref: https://github.com/golang/lint/issues/68
-  failedLint=$(ls "$p"/*.go | egrep -v "(zz_generated.*.go|generated.pb.go|generated.proto|types_swagger_doc_generated.go)" | xargs -L1 golint 2>/dev/null)
+  failedLint=$(echo "$p"/*.go | grep -E -v "(zz_generated.*.go|generated.pb.go|generated.proto|types_swagger_doc_generated.go)" | xargs -L1 golint 2>/dev/null)
```

Comment:
```
In ./hack/verify-golint.sh line 66:
  failedLint=$(ls "$p"/*.go | egrep -v "(zz_generated.*.go|generated.pb.go|generated.proto|types_swagger_doc_generated.go)" | xargs -L1 golint 2>/dev/null)
               ^----------^ SC2012: Use find instead of ls to better handle non-alphanumeric filenames.
                              ^---^ SC2196: egrep is non-standard and deprecated. Use grep -E instead.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2012
```